python_ta/transforms/setendings.py

```diff
@@ -41,63 +63,36 @@
 NODES_WITH_CHILDREN = [
     astroid.Assert,
     astroid.Assign,
-    # TODO: This one identifies only the expression to the left of the period,
-    # and not the name of the attribute.
-    # Given 'self.name = 10', it will highlight 'self' rather than 'self.name'
-    astroid.AssignAttr,
-    # TODO: Include the 'async' keyword in expressions for all Async* nodes.
     astroid.AsyncFor,
     astroid.AsyncFunctionDef,
     astroid.AsyncWith,
-    # TODO: Same problem as AssignAttr (attribute missing)
-    astroid.Attribute,
     astroid.AugAssign,
     astroid.Await,
     astroid.BinOp,
     astroid.BoolOp,
-    # TODO: missing right parens
     astroid.Call,
     astroid.ClassDef,
     astroid.Compare,
-    # TODO: missing right parens (note: only if decorator takes args)
+    astroid.Comprehension,
     astroid.Decorators,
-    # TODO: missing keyword 'del' and attribute name
-    astroid.DelAttr,
     astroid.Delete,
-    # TODO: missing right }
-    astroid.Dict,
-    # TODO: missing right }
-    astroid.DictComp,
     astroid.ExceptHandler,
-    # TODO: missing *both* outer brackets
     astroid.ExtSlice,
-    # TODO: missing right paren
-    astroid.Expr,
+    # astroid.Expr,  # need this here?
     astroid.For,
     astroid.FunctionDef,
-    # TODO: missing *both* outer parens
     astroid.GeneratorExp,
+    
     # TODO: need to fix elif (start) col_offset
     astroid.If,
     astroid.IfExp,
-    # TODO: missing *both* outer brackets
     astroid.Index,
-    # TODO: would be good to see the name of the keyword as well
     astroid.Keyword,
```

Comment:
This is needed here, for example when the `end_lineno` is set from its `Const` child.